app/uk/gov/hmrc/agentclientrelationships/connectors/HIPConnector.scala

```diff
@@ -153,6 +162,38 @@ class HIPConnector @Inject() (
       }
   }
 
+  // HIP API #EPID1521 Create/Update Agent Relationship //url and error handling is different to getActiveClientRelationships
+  override def getAllRelationships(
+    taxIdentifier: TaxIdentifier
+  )(implicit
+    hc: HeaderCarrier,
+    ec: ExecutionContext
+  ): Future[Either[RelationshipFailureResponse, Seq[ClientRelationship]]] = {
+    val url = getActiveClientRelationshipsHipUrl(taxIdentifier)
+
+    implicit val reads: Reads[ClientRelationship] = ClientRelationship.hipReads
+
+    EitherT(getRelationship(s"GetActiveClientRelationships", url))
+      .map(response => (response.json \ "relationshipDisplayResponse").as[Seq[ClientRelationship]])
+      .leftMap[RelationshipFailureResponse] { errorResponse =>
+        errorResponse.statusCode match {
+          case Status.NOT_FOUND => RelationshipFailureResponse.RelationshipNotFound
+          case Status.UNPROCESSABLE_ENTITY if errorResponse.getMessage.contains("suspended") =>
+            RelationshipFailureResponse.RelationshipSuspended
```

Comment:
These two error cases immediately become empty list, can't we return Nil at the connector level to save on handling them?

Reply:
Yeah that makes sense, was more so thinking that if there hasn't been a need to handle the other responses since we've had the original call set up that likely won't change. At least for the notfound/suspended cases which we always treat the same.